content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
+and to advocate for the inclusion of these marginalized groups in the
+computer science community at large.
+
+## Representation ##
+
+The Women in Computer Science Undergraduate Subcommittee aims to
+represent and support undergraduate cis and trans women, and non-binary
+individuals in computer science. Additionally, we seek to include and
+support members of all underrepresented groups in the fields of
+technology and computing, with the most varied and diverse backgrounds
+possible.
+
+## Values ##
+
+1.  Inclusiveness: The undergraduate commitee is committed to creating
+    and inclusive environment in which women of all backgrounds will
+    feel welcome.
+
+2.  Humility: In all it's activities, the committee will acknowledge and
+    respect the perspectives of those with experiences different to
+    ours, and will strive to understand and incorporate these
+    perspectives.
+
+3.  Respoect: All members of the community will be respected, despite
+    difference in opinions and values.
+
+4.  Transparency: The actions of the committee will be visible to
+    promote accountability, and all decision-making will be open and
+    well-documented.
+
+5.  Honesty: The committee will create an environment to promote honest
+    disclosure. All members should feel free speaking about any opinions
+    and feelings.
+
+6.  Technical Excellence: The committee acknowledges that it's
+    fundamental goal is to support diversity in technical work, and
+    members will strive to achieve excellence in this work themselves.
+
+# Committee Composition #
+
+1.  Only undergraduate cis- and trans-women and non-binary individuals
+    enrolled in computer science or a related program may serve on the
```

Comment:
I think it should be mentioned explicitly to avoid SE people feeling "left out". Traditionally, SE has been overrepresented on the committee so I don't see any harm in listing them.